iquip/apps/builder.py

```diff
@@ -587,3 +604,33 @@ def onSubmitted(self, rid: int):
     def frames(self) -> Tuple[Tuple[str, BuilderFrame]]:
         """Overridden."""
         return (("", self.builderFrame),)
+
+class _ScanEntry(QWidget):
```

Comment:
Please add one more empty line before the class definition.
And we wrote all entry classes in the above part, thus I think this should be there.

Reply:
I missed it.
I will move _ScanEntry to the upper part.